brainstormforce/ultimate-addons-for-gutenberg
# fix: Skipped parsing large json file & converted it to php file.

### Description
We were json_decode the json file which was almost 3mb & due to which if user's site has low memory the it would crash.
So we converted that JSON to php array & used it.

### Types of changes
 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
 - Check Icon Picker in Editor.
 - Check Icon Rendering in Editor
 - Check Icon rendering on the frontend.

### Checklist:
